assembly/pallets/timestamp/assembly/timestamp.ts

```diff
@@ -54,44 +56,35 @@ export class Timestamp {
      * @param now timestamp number
      */
     static set(now: Moment): u8[] {
```

Comment:
I think that this should not be exported outside the runtime?

Reply:
It is exported in Substrate node-template metadata

Reply:
Let's comment this on the daily